core/src/Kokkos_Core.hpp

```diff
@@ -81,14 +81,16 @@ struct InitArguments {
   int ndevices;
   int skip_device;
   bool disable_warnings;
-
-  InitArguments(int nt = -1, int nn = -1, int dv = -1, bool dw = false)
+  bool tune_kokkos_internals;
```

Comment:
Why did you specify "kokkos" in the name?

Reply:
It's just how I think of it, this is describing whether Kokkos tunes its internals. I don't feel too strongly about it

Reply:
I commented because it feels inconsistent to me next to `g_tune_internals` and to `tune_internals()`

Reply:
Works for me, pushed 9c8023d